lib/config.js

```diff
@@ -48,7 +48,12 @@ config.mongodb.connectOptions = {
     journal: true
   },
   server: {
-    auto_reconnect: true
+    socketOptions: {
+      autoReconnect: true, // reconnect on connection failure
+      keepAlive: 15000, // TCP keep-alive every 15 seconds (useful for VPN'd connections)
+    },
+    reconnectInterval: 1000, // attempt reconnect every second
```

Comment:
The default is already 1000 for this ... so we should probably not specify this here?